front/src/components/BackOfficeModal.jsx

```diff
@@ -0,0 +1,53 @@
+import routes from "@/routes"
+import { CheckCircleIcon, PencilIcon, TrashIcon, XCircleIcon } from "@heroicons/react/24/outline"
+import Link from "next/link"
+import { useState } from "react"
+
+// eslint-disable-next-line no-unused-vars
```

Comment:
why ?

Reply:
it was just to place de id for the futur